pkgs/os-specific/linux/systemd/default.nix

```diff
@@ -24,8 +24,8 @@ stdenv.mkDerivation rec {
   src = fetchFromGitHub {
     owner = "NixOS";
     repo = "systemd";
-    rev = "aa4c4d39d75ce52664cb28d569b1ceafda7b4c06";
-    sha256 = "1ax94gzbdwdcf3wgj7f9cabdkvn2zynnnli7gkbz4isidlpis86g";
+    rev = "6b8d75f093a0e05d9001398c81028c4b688ef801";
```

Comment:
Please update to https://github.com/NixOS/systemd/commit/5fb35fbc783516e2014115c3488134a2afb8494c